pkg/openscap/openscap.go

```diff
@@ -0,0 +1,193 @@
+package openscap
+
+import (
+	"fmt"
+	docker "github.com/fsouza/go-dockerclient"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"syscall"
+)
+
+const (
+	CPE             = "oval:org.open-scap.cpe.rhel:def:"
+	CPEDict         = "/usr/share/openscap/cpe/openscap-cpe-oval.xml"
+	CVEUrl          = "https://www.redhat.com/security/data/metrics/ds/"
+	DistCVENameFmt  = "com.redhat.rhsa-RHEL%d.ds.xml.bz2"
+	ArfResultFile   = "results-arf.xml"
+	TmpDir          = "/tmp"
+	Linux           = "Linux"
+	OpenSCAP        = "OpenSCAP"
+	ImageShortIDLen = 11
+)
+
+var (
+	RHELDistNumbers = [...]int{5, 6, 7}
+)
+
+// Scanner is the interface of OpenSCAP scanner
+type Scanner interface {
```

Comment:
@pweil- I am not sure in which package: because inspector package is importing openscap i can't put this declaration in inspector and I don't think that the correct location is in openscap. maybe I should make a new package just for scanner and make openscap a sub package of that? I am not sure what is the correct structure here.